src/EditorFeatures/Core.Wpf/InlineDiagnostics/AbstractDiagnosticsTaggerProvider.SingleDiagnosticKindPullTaggerProvider.cs

```diff
@@ -128,7 +128,6 @@ private async Task ProduceTagsAsync(
                     document,
                     requestedSpan.Span.ToTextSpan(),
                     diagnosticKind: _diagnosticKind,
-                    includeSuppressedDiagnostics: true,
```

Comment:
Note: if a featuree passes `includeSuppressedDiagnostics: true`, it needs no change.  We always return all diagnostics by default, including suppressed ones.